custom-recipes/api-connect/recipe.json

```
@@ -193,7 +198,7 @@
             "description": "",
             "type": "STRING",
             "defaultValue": null,
-            "visibilityCondition": "(['page', 'offset'].indexOf(model.pagination_type)>-1)"
+            "visibilityCondition": "(['offset'].indexOf(model.pagination_type)>-1)"
```

Maybe change it as this parameter is now only her for what pagination_type
```suggestion
            "visibilityCondition": "model.pagination_type == 'offset'"
```